### PR TITLE
[doc] Upgrade README.md file in test folder.

### PR DESCRIPTION
**Pull request by BruceDai**
@ibelem PTAL, thanks

### CODE REVIEW

File: test/README.md [hunk 1]
```diff
@@ -1,7 +1,29 @@
-# Contribute CTS test case
+# Test Documentation
+## Introduction
+This **test** folder contains test cases and relative assistant tools for testing **WebNN API**.
+
+According to [API Documentation](https://github.com/intel/webml-polyfill/blob/master/docs/api.md) and [API Info](https://github.com/intel/webml-polyfill/blob/master/src/nn/Enums.js), these test cases including:
+* **Unit Tests** in `base` folder and `unit-test` folder, checking Interface, Attribute and Method to ensure meet its design and behaves as intended,
+* **CTS(Compatibility Test Suite) Tests** in `cts` folder, focus on testing behaves of [supported ops](https://github.com/intel/webml-polyfill/blob/master/docs/supported_ops.md), checking compatibility on cross platforms,
```
Comment (ibelem) [inline test/README.md]: /CTS(Compatibility Test Suite)/CTS (Compatibility Test Suite)/g + others

File: test/README.md [hunk 2]
```diff
@@ -1,7 +1,29 @@
-# Contribute CTS test case
+# Test Documentation
+## Introduction
+This **test** folder contains test cases and relative assistant tools for testing **WebNN API**.
+
+According to [API Documentation](https://github.com/intel/webml-polyfill/blob/master/docs/api.md) and [API Info](https://github.com/intel/webml-polyfill/blob/master/src/nn/Enums.js), these test cases including:
+* **Unit Tests** in `base` folder and `unit-test` folder, checking Interface, Attribute and Method to ensure meet its design and behaves as intended,
```
Comment (ibelem) [inline test/README.md]: Add link for these tests

Comment (ibelem) [inline test/README.md]: why the "," show in end of bullets

File: test/README.md [hunk 3]
```diff
@@ -1,7 +1,29 @@
...
+* **CTS(Compatibility Test Suite) Tests** in `cts` folder, focus on testing behaves of [supported ops](https://github.com/intel/webml-polyfill/blob/master/docs/supported_ops.md), checking compatibility on cross platforms,
+* **End-to-End Tests** in `end2end-test` folder, testing using [ONNX models and sample data](https://github.com/intel/webml-polyfill/blob/master/test/end2end-test/README.md), checking model flow being performed as intended,
+* **Real Model Tests** in `realmodel` folder, testing behaves of some supported ops in a [real-model](https://github.com/intel/webml-polyfill/tree/master/test/realmodel#3-supported-onnxjs-models) scenario.
+```
+Note: Above 4 type tests were wrote with [Mocha test framework](https://mochajs.org/).
+```
+* **IDL(Interface Definition Language) Tests** in `wpt` folder which were generated by [idlharness.js API](https://web-platform-tests.org/writing-tests/idlharness.html).
+
+These relative assistant tools:
```
Comment (ibelem) [inline test/README.md]: /These relative assistant tools:/Assistant tools:/

File: test/README.md [hunk 4]
```diff
@@ -1,7 +1,29 @@
...
+* **Accuracy Measurement** in `tool/MeasureAccuracy` folder, for checking accuracy of semantic segmentation inference result with inputing lots of images,
+* **CI(Continuous Integration)**  its baseline data in `tool/CI` folder, for checking webml-polyfill repo's PRs, leveraging [Travis CI](https://travis-ci.com/) tool testing on macOS platform, [Circle CI](https://circleci.com/) tool testing on Linux platform and [AppVeyor CI](https://www.appveyor.com/) on Windows platform,
```
Comment (ibelem) [inline test/README.md]: its baseline data in `tool/CI` folder - it's not a comprehensive sentence

File: test/README.md [hunk 5]
```diff
@@ -1,7 +1,29 @@
...
+* **per Layer Analyzer** in `tool/perLayerAnalyzer` folder, for running **Real Model Tests**, collecting and displaying each layer's performance to help analysing,
+* **Regression Checker** in `tool/RegressionChecker` folder, for checking native backend implement PRs with high quality(avoiding new problems such as crash, freeze, etc.).
+
+
+## Contribute CTS test case
 The CTS tests in `./cts/test` folder were converted from original Android CTS test, for new created CTS test file, please add them into `./cts/test_supplement` folder
 
-## Steps
+### Steps
 Here are steps to update `./cts/test_supplement/cts_supplement-all.js`:
```
Comment (ibelem) [inline test/README.md]: /Here are steps to update/Steps to update/

**Comment by BruceDai** [comment]
@ibelem Please review updated pr, thanks

**Comment by ibelem** [comment]
Thanks @BruceDai, LGTM

**Pull request merged**